src/main/flight/pid.c

```diff
@@ -1171,12 +1164,12 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, timeUs_t currentTim
 
         // P boost at the end of throttle chop
         // attenuate effect if turning more than 50 deg/s, half at 100 deg/s
-        float agBoostAttenuator = fabsf(currentPidSetpoint) / 50.0f;
-        agBoostAttenuator = MAX(agBoostAttenuator, 1.0f);
-        const float agBoost = 1.0f + (pidRuntime.antiGravityPBoost / agBoostAttenuator);
+        float agSetpointAttenuator = fabsf(currentPidSetpoint) / 50.0f;
+        agSetpointAttenuator = MAX(agSetpointAttenuator, 1.0f);
+        const float antiGravityPBoost = 1.0f + (pidRuntime.antiGravityThrottleD / agSetpointAttenuator) * pidRuntime.antiGravityPGain;
+        DEBUG_SET(DEBUG_ANTI_GRAVITY, 3, lrintf(antiGravityPBoost * 100));
```

Comment:
should this be wrapped with "if (axis == FD_ROLL)", otherwise the same value being set three times for all three axis RPY

Reply:
Good point!  And since the calculations only apply for roll and pitch, they can be ignored if axis is yaw.  Thanks!